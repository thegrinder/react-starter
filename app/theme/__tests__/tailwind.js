import tailwind from '../tailwind';

describe('tailwind theme', () => {
  it('should be defined', () => {
    expect(tailwind).toBeDefined();
    expect(tailwind).toMatchSnapshot();
  });
});
