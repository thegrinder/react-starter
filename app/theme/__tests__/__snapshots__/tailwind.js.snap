// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tailwind theme should be defined 1`] = `
Object {
  "borderRadius": Object {
    "0": "0",
    "1": ".125rem",
    "2": ".25rem",
    "3": ".5rem",
    "4": "1rem",
    "pill": "9999px",
  },
  "fontWeights": Object {
    "4": 400,
    "7": 700,
  },
  "height": Object {
    "10": "2.5rem",
    "12": "3rem",
    "16": "4rem",
    "24": "6rem",
    "32": "8rem",
    "4": "1rem",
    "48": "12rem",
    "6": "1.5rem",
    "64": "16rem",
    "8": "2rem",
    "auto": "auto",
    "full": "100%",
    "screen": "100vh",
  },
  "leading": Object {
    "loose": 2,
    "none": 1,
    "normal": 1.5,
    "tight": 1.25,
  },
  "maxHeight": Object {
    "full": "100%",
    "screen": "100vh",
  },
  "maxWidth": Object {
    "full": "100%",
    "lg": "50rem",
    "md": "40rem",
    "sm": "30rem",
    "xl": "60rem",
    "xs": "20rem",
  },
  "minHeight": Object {
    "0": "0",
    "full": "100%",
    "screen": "100vh",
  },
  "minWidth": Object {
    "0": "0",
    "full": "100%",
  },
  "modules": Object {
    "appearance": false,
    "backgroundAttachment": false,
    "backgroundColors": false,
    "backgroundPosition": false,
    "backgroundRepeat": false,
    "backgroundSize": false,
    "borderColors": false,
    "borderRadius": false,
    "borderStyle": false,
    "borderWidths": false,
    "cursor": false,
    "display": false,
    "flexbox": Array [
      "responsive",
    ],
    "float": false,
    "fontWeights": false,
    "fonts": false,
    "height": Array [],
    "leading": Array [],
    "lists": false,
    "margin": false,
    "maxHeight": Array [],
    "maxWidth": Array [],
    "minHeight": Array [],
    "minWidth": Array [],
    "negativeMargin": false,
    "opacity": false,
    "overflow": Array [],
    "padding": false,
    "pointerEvents": false,
    "position": Array [],
    "resize": false,
    "shadows": false,
    "svgFill": false,
    "svgStroke": false,
    "textAlign": false,
    "textColors": false,
    "textSizes": false,
    "textStyle": false,
    "tracking": Array [],
    "userSelect": false,
    "verticalAlign": false,
    "visibility": false,
    "whitespace": false,
    "width": Array [],
    "zIndex": Array [],
  },
  "opacity": Object {
    "0": "0",
    "100": "1",
  },
  "options": Object {
    "important": false,
    "prefix": "",
    "separator": ":",
  },
  "plugins": Array [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
  "screens": Object {
    "lg": "992px",
    "md": "768px",
    "sm": "576px",
    "xl": "1200px",
  },
  "spacing": Object {
    "0": "0",
    "1": "0.25rem",
    "12": "3rem",
    "16": "4rem",
    "2": "0.5rem",
    "24": "6rem",
    "3": "0.75rem",
    "32": "8rem",
    "4": "1rem",
    "48": "12rem",
    "6": "1.5rem",
    "64": "16rem",
    "8": "2rem",
    "auto": "auto",
  },
  "width": Object {
    "10": "2.5rem",
    "12": "3rem",
    "16": "4rem",
    "24": "6rem",
    "32": "8rem",
    "4": "1rem",
    "48": "12rem",
    "6": "1.5rem",
    "64": "16rem",
    "8": "2rem",
    "auto": "auto",
    "full": "100%",
  },
  "zIndex": Object {
    "0": 0,
    "1": 1,
    "2": 2,
  },
}
`;
